src/edr_pydantic/edr_feature_collection.py

```diff
@@ -0,0 +1,9 @@
+from typing import Dict
+
+from edr_pydantic.base_model import EdrBaseModel
+from edr_pydantic.parameter import Parameter
+from geojson_pydantic import FeatureCollection  # type: ignore
```

Comment:
Why do we need a `# type: ignore` here?

Reply:
Because mypy is not able to find the stubs for the external packages. I would have to look into it, but I think that there are two possible fixes:

1. add `py.typed` to the library and ensure it is distributed by adding it to pytoml or setup.py:
```
[tool.setuptools.package-data]
"<package_name>" = ["py.typed"]
```

2. Ignore it.